packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -27,35 +36,40 @@ const defaultValue = 'This is a caption';
 const groupIdentifier = 'CAPTION VARIANTS';
 
 <Caption>
-  {select(label, LANGUAGE_VARIANTS, defaultValue, groupIdentifier)}
+  {select(label, LANGUAGE_VARIANTS, LANGUAGE_VARIANTS.English, groupIdentifier).text}
 </Caption>
 ```
 
 ### inputProvider
 
-N.B. the number of elements in the array should equal the number of arguments taken
-by the render function.
-
 ```jsx
+import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import Caption from '@bbc/psammead-caption';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 
 storiesOf('Caption', module)
   .addDecorator(withKnobs)
   .add(
     'default',
     inputProvider(
-      ['caption', 'visually hidden text'],
-      (captionText, vhText) => (
-        <Caption>
-          <VisuallyHiddenText>{vhText}</VisuallyHiddenText>
+      [
+        { name: 'caption', defaultText: 'Students sitting an examination' },
+        { name: 'offscreen text', defaultText: 'Image Caption, ' },
+      ],
+      ([captionText, offscreenText], script, dir) => (
+        <Caption script={script} dir={dir}>
+          <VisuallyHiddenText script={script} dir={dir}>
```

Comment:
tiny thing, `VisuallyHiddenText` is hidden so wouldn't have a dir :P

Reply:
Nor a `script` :+1: